bert_e/tests/test_bert_e.py

```diff
@@ -2472,6 +2472,413 @@ def test_bypass_options(self):
+        settings = """
+repository_owner: {owner}
+repository_slug: {slug}
+repository_host: {host}
+robot: {robot}
+robot_email: nobody@nowhere.com
+pull_request_base_url: https://bitbucket.org/{owner}/{slug}/bar/pull-requests/{{pr_id}}
+commit_base_url: https://bitbucket.org/{owner}/{slug}/commits/{{commit_id}}
+build_key: pre-merge
+need_author_approval: True
+required_leader_approvals: 1
+required_peer_approvals: 2
+project_leaders:
+  - {admin}
+pr_author_options:
+  {contributor}:
+    - bypass_author_approval
+""" # noqa
+        pr = self.create_pr('feature/TEST-0043', 'development/10.0')
+        self.handle(pr.id, settings=settings)
+        self.assertIs(len(list(pr.get_comments())), 2)
+        self.assertIn('bypass_author_approval', self.get_last_pr_comment(pr))
+
+        settings = """
+repository_owner: {owner}
+repository_slug: {slug}
+repository_host: {host}
+robot: {robot}
+robot_email: nobody@nowhere.com
+pull_request_base_url: https://bitbucket.org/{owner}/{slug}/bar/pull-requests/{{pr_id}}
+commit_base_url: https://bitbucket.org/{owner}/{slug}/commits/{{commit_id}}
+build_key: pre-merge
+need_author_approval: True
+required_leader_approvals: 1
+required_peer_approvals: 2
+project_leaders:
+  - {admin}
+pr_author_options:
+  {contributor}:
+    - bypass_peer_approval
+""" # noqa
+        pr = self.create_pr('feature/TEST-0044', 'development/10.0')
+        self.handle(pr.id, settings=settings)
+        self.assertIs(len(list(pr.get_comments())), 2)
+        self.assertIn('bypass_peer_approval', self.get_last_pr_comment(pr))
+
+        settings = """
+repository_owner: {owner}
+repository_slug: {slug}
+repository_host: {host}
+robot: {robot}
+robot_email: nobody@nowhere.com
+pull_request_base_url: https://bitbucket.org/{owner}/{slug}/bar/pull-requests/{{pr_id}}
+commit_base_url: https://bitbucket.org/{owner}/{slug}/commits/{{commit_id}}
+build_key: pre-merge
+need_author_approval: True
+required_leader_approvals: 1
+required_peer_approvals: 2
+project_leaders:
+  - {admin}
+pr_author_options:
+  {contributor}:
+    - bypass_build_status
+""" # noqa
+        pr = self.create_pr('feature/TEST-0045', 'development/10.0')
+        self.handle(pr.id, settings=settings)
+        self.assertIs(len(list(pr.get_comments())), 2)
+        self.assertIn('bypass_build_status', self.get_last_pr_comment(pr))
+
+    def test_bypass_author_jira(self):
+        settings = """
+repository_owner: {owner}
+repository_slug: {slug}
+repository_host: {host}
+robot: {robot}
+robot_email: nobody@nowhere.com
+pull_request_base_url: https://bitbucket.org/{owner}/{slug}/bar/pull-requests/{{pr_id}}
+commit_base_url: https://bitbucket.org/{owner}/{slug}/commits/{{commit_id}}
+build_key: pre-merge
+need_author_approval: True
+required_leader_approvals: 1
+required_peer_approvals: 2
+project_leaders:
+  - {admin}
+pr_author_options:
+  {contributor}:
+    - bypass_build_status
+    - bypass_peer_approval
+    - bypass_jira_check
+"""  # noqa
```

Comment:
I don't see any jira config inside this settings to ensure that bert-e is checking jira. Is it set elsewhere? On top of that we're not check if the `self.handle` is a success or not.